quickstarts/spring-boot-2-camel-rhosak-template.json

```diff
@@ -0,0 +1,612 @@
+    "template": "s2i-fuse79-spring-boot-2-camel-rhosak"
+  },
+  "parameters": [
+    {
+      "name": "APP_NAME",
+      "displayName": "Application Name",
+      "required": true,
+      "value": "s2i-fuse79-sb2-rhosak",
+      "description": "The name assigned to the application (max 20 characters)."
+    },
+    {
+      "name": "GIT_REPO",
+      "displayName": "Git Repository URL",
+      "required": true,
+      "value": "https://github.com/jboss-fuse/spring-boot-camel-rhosak.git",
+      "description": "The URL of the repository with your application source code."
+    },
+    {
+      "name": "GIT_REF",
+      "displayName": "Git Reference",
+      "value": "master",
+      "description": "Set this to a branch name, tag or other ref of your repository if you are not using the default branch."
+    },
+    {
+      "name": "BROKER_HOST",
+      "displayName": "A-MQ Broker Service",
+      "required": true,
+      "value": "broker-amq-tcp",
+      "description": "Set this to the name of the TCP service of the A-MQ broker. You may need to create a broker first."
+    },
+    {
+      "name": "BROKER_USERNAME",
+      "displayName": "A-MQ Broker Username",
+      "required": true,
+      "description": "The username used to authenticate with the A-MQ broker."
+    },
+    {
+      "name": "BROKER_PASSWORD",
+      "displayName": "A-MQ Broker Password",
+      "required": true,
+      "description": "The password used to authenticate with the A-MQ broker."
+    },
+    {
+      "name": "DATABASE_HOST",
+      "displayName": "PostgreSQL Service",
+      "required": true,
+      "value": "postgresql",
+      "description": "Set this to the name of the PostgreSQL service. You may need to create the database first."
+    },
+    {
+      "name": "DATABASE_USERNAME",
+      "displayName": "PostgreSQL Username",
+      "required": true,
+      "description": "The username used to authenticate with the PostgreSQL database."
+    },
+    {
+      "name": "DATABASE_PASSWORD",
+      "displayName": "PostgreSQL Password",
+      "required": true,
+      "description": "The password used to authenticate with the PostgreSQL database."
+    },
+    {
+      "name": "VOLUME_SIZE",
+      "displayName": "Volume size for transaction log",
+      "required": true,
+      "value": "1Gi",
+      "description": "The size of the volume used to store the transaction log."
+    },
+    {
+      "name": "BASE_IMAGE_NAME",
+      "displayName": "Base Image Name, JDK11 or JDK8",
+      "value": "fuse7-java-openshift",
+      "description": "The base image used. Default is fuse7-java-openshift which is JDK8 based; fuse7-java11-openshift is JDK11 based"
+    },
+    {
+      "name": "BUILDER_VERSION",
+      "displayName": "Builder version",
+      "value": "1.9",
+      "description": "The version of the Fuse S2I builder image to use."
+    },
+    {
+      "name": "APP_VERSION",
+      "displayName": "Application Version",
+      "value": "2.2.64-SNAPSHOT",
```

Comment:
Is it expected to have a slightly different version than in the other template? https://github.com/jboss-fuse/application-templates/pull/184/files#r657885441

Reply:
This is okay as well, will be build time aligned with the version of the quickstart built.